docs/flink/flink-getting-started.md

```diff
@@ -526,6 +526,27 @@ FlinkSink.forRowData(input)
 env.execute("Test Iceberg DataStream");
 ```
 
+## Write options
+
+Flink write options are passed when configuring the FlinkSink, like this:
+
+```
+FlinkSink.Builder builder = FlinkSink.forRow(dataStream, SimpleDataUtil.FLINK_SCHEMA)
+    .table(table)
+    .tableLoader(tableLoader)
+    .set("write-format", "orc")
+    .set(FlinkWriteOptions.OVERWRITE_MODE, "true");
+```
+
+| Flink option           | Default                    | Description                                                  |
+| ---------------------- | -------------------------- | ------------------------------------------------------------ |
+| write-format           | Table write.format.default | File format to use for this write operation; parquet, avro, or orc |
+| target-file-size-bytes | As per table property      | Overrides this table's write.target-file-size-bytes          |
+| write.upsert.enabled | Table write.upsert.enabled | Overrides this table's write.upsert.enabled |
+| overwrite | false | Overwrite the table's data |
+| write.distribution-mode | Table write.distribution-mode | Overrides this table's write.distribution-mode |
```

Comment:
I don't think that the `write.` prefix makes sense to have here. This is configuring a write so it is implied. This should just be `distribution-mode`.